src/web/server/htmlTemplate.ts

```diff
@@ -84,18 +84,24 @@ export const htmlTemplate = ({
             )}" as="font" crossorigin>`,
     );
 
-    const generateMetaTags = (dataObject: { [key: string]: string }) => {
+    const generateMetaTags = (
+        dataObject: { [key: string]: string },
+        attributeName: 'name' | 'property',
```

Comment:
not sure whether this is the idiomatic way to do it...

Reply:
Sometimes I upgrade to:

```
const generateMetaTags = ({
        dataObject,
        attributeName,
}: {
        dataObject: { [key: string]: string },
        attributeName: 'name' | 'property',
}) => ...
```

To guard against property order confusion but it's more verbose. I think you can go both ways here.